manifests/init.pp

```diff
@@ -158,6 +158,7 @@
   Array $remote_read_configs  = $::prometheus::params::remote_read_configs,
   Array $remote_write_configs = $::prometheus::params::remote_write_configs,
   $alerts                     = $::prometheus::params::alerts,
+  $extra_alerts               = {},
```

Comment:
Please add a datatype for this new parmeter and documentation in the header.

Reply:
Ok, i will divert from the existing style :-)